reactnative/README.md

```diff
@@ -0,0 +1,66 @@
+# React Native Input Mask
+
+### This is beta and is a still a WIP!
+
+## Getting started
+
+First, install it.
+
+```bash
+npm i react-native-text-mask --save
```

Comment:
Nah. I know `react-native` is a lot but it's the standard naming convention of all RN libraries.